source/cs/0.3.0/index.html.haml

```diff
@@ -0,0 +1,177 @@
+  
+
+  ### Jak mohu vyžadovanou snahu snížit na minimum?
+  Vždycky měj nahoře sekci `"Unreleased"`, ve které budou schromažďovány všechny 
+  změny. 
+
+  To plní hned dva účely:
+
+  - Lidé uvidí, jaké změny mohou očekávat v následujících vydáních
+  - V momentu vydání stačí změnit `"Unreleased"` na číslo verze a přidat nový nadpis `"Unreleased"` na vrch dokumentu.
+
+  ### Co zaručeně rozbrečí jednorožce?
+  Dobrá… Tak si to povíme.
+  
+  - **Zkopírování diffu commit logů.** To prostě nedělej, nikomu to nepomůže.
+  - **Nezdůraznění dále nepodporovaných funcí.** Když lidé aktualizují na novou verzi, mělo by jim být bolestivě jasné, že se něco rozbije.
+  - **Data v regionálních formátech.** V USA píšou lidé nejdříve měsíc ("06-02-2012" pro 2. června 2012, což nedává *vůbec žádný* smysl), zatímco mnoho lidí ve zbytku světa píše roboticky vypadající verzi "2 June 2012", ale vyslovují to jinak. "2012-06-02" je logicky napsané od největšího po nejmenší celek, nepřekrývá se nesrozumitelně s jinými fomáty data a je to [ISO standard](http://www.iso.org/iso/home/standards/iso8601.htm). Proto je to doporučovaný formát dat pro changelogy. 
+  
+  To ale není všechno. Pomoz mi sbírat jednorožčí slzy tím že 
+  [otevřeš issue][issues] nebo pull request.
+  
+  ### Existuje pro formát changelogů nějaký standard?
+  Bohužel ne. Klid. Vím, že se zuřivě snažíš najít ten odkaz na GNU návod jakým 
+  stylem psát changelog, nebo onen dvouodstavcový GNU NEWS soubor, který si říká 
+  "směrnice". Zmíněný GNU návod je sice hezký začátek, ale je smutně naivní. Být
+  naivní není nic špatného, ale když lidé potřebují radu, málokdy to někomu 
+  pomůže. Obzvlášť, když existuje tolik situací a okrajových případů, se kterými
+  se musí člověk nějak poprat. 
+  
+  Tento projekt 
+  [obsahuje něco, co se doufám jednou stane lepší konvencí pro CHANGELOGy][CHANGELOG].
+  Nemyslím si, že je momentální stav dostatečně dobrý, a jsem toho názoru, že 
+  jsme jako komunita schopni vymyslet lepší konvence, pokud se budeme snažit 
+  vybrat ty nejlepší praktiky z doopravdových softwarových projektů. 
+  Porozhlédněte se a mějte na paměti, že 
+  [diskuse a návrhy na zlepšení jsou vítány][issues]!
+  
+  ### Jak by se měl changelog jmenovat?
+  Inu, pokud jste to už nepoznali z příkladu výše, `CHANGELOG.md` je zatím ta 
+  nejlepší konvence. 
+  
+  Některé projekty používají `HISTORY.txt`, `HISTORY.md`, `History.md`, 
+  `NEWS.txt`, `NEWS.md`, `News.txt`, `RELEASES.txt`, `RELEASE.md`, 
+  `releases.md`, apod.
+
+  Je v tom binec. Všecha tato jména jen komplikují život lidem, kteří se ten 
+  dokument snaží najít. 
+  
+  ### Proč lidé jednoduše nepoužívají `git log` diff?
+  Protože diffy logů jsou plné šumu — přirozeně. Nebyly by vyhovujícím 
+  changelogem ani v hypotetickém projektu vedeném dokonalými lidmi, kteří nikdy
+  nedělají překlepy, nikdy nezapomínají commitnout nové doubor a nikdy jim 
```

Comment:
Typo in the line 107; it should state: `nedělají překlepy, nikdy nezapomínají commitnout nové soubory a nikdy jim`